docs/jotai/api/babel.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Babel
+description: This doc describes the `jotai/babel` bundle.
+nav: 2.4
+---
```

Comment:
Doesn't look like formatting is well supported in the current description component.

![image](https://user-images.githubusercontent.com/23324155/133317368-7d4ad9fb-3ac1-496b-a4af-c2054db6d789.png)

I'll see if I can run a fix by.

Reply:
Fixed this in db71847346ee95836b230afac89633f1676d1532. Carry on 😅.